src/UI/Launcher.module.css

```diff
@@ -26,3 +26,9 @@
 	height: 35px;
 	padding: 10px 0 10px 0; /* Makes sure the img is centered correctly in the round button */
 }
+
+.launcher.state-open {
+}
+
+.launcher.state-minimize {
```

Comment:
launcher moet er volgends mij uit.

en hoort het leeg?

Reply:
Het hoort leeg, is een css class die we zetten om anderen css op te kunnen laten toevoegen.

Reply:
zie https://github.com/parley-messaging/web-library/pull/58#discussion_r1574256654